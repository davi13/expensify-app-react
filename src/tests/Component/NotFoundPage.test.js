import React from 'react'
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';
import expectExport from 'expect';

test('Should render not found page', () => {
    const wrapper = shallow(<NotFoundPage />);
    expectExport(wrapper).toMatchSnapshot();
})