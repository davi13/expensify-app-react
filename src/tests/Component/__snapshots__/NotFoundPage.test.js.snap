// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render not found page 1`] = `<wrapper />`;
